script.js

```diff
@@ -260,19 +264,33 @@ function startAlgo(event) {
   } 
   startLooping();
 }
-
+function fillTable() {
+  table = document.getElementById("myTable");
+  var row = table.insertRow(counter);
+  counter++;
+  var cell1 = row.insertCell(0);
+  var cell2 = row.insertCell(1);
+  var cell3 = row.insertCell(2);
+  var cell4 = row.insertCell(3);
+  var cell5 = row.insertCell(4);
+
+  cell1.innerHTML = a;
+  cell2.innerHTML = b;
+  cell3.innerHTML = midpoint;
+  cell4.innerHTML = f(midpoint) * f(a);
+  cell5.innerHTML = f(midpoint) * f(b);
+  console.log (a + " " + b + " " + midpoint + " " + f(midpoint) * f(a) + " " + f(midpoint) * f(b) );
+}
 function startLooping() {
   var finish = false;
-
   midpoint = (a + b) / 2;
+  fillTable();
```

Comment:
The counter idea is not that good, actually I used the lineRange count on line 302 to get the number of iteration at the end, but I don't think this is a good idea, and it might be better if we passed a variable accross the recursive function called.
so startLooping(stepNumber)
and then pass this number to fillTable(stepNumber)
and also print the number of steps at the end using the same number.
What do you think?